package.json

```diff
@@ -58,6 +58,7 @@
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^12.1.5",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
We do not use this library in our test. We use our own test-utils to manipulate with the elements